core/domain/takeout_service_test.py

```diff
@@ -388,6 +388,8 @@ def test_export_data_trivial(self):
     def test_export_data_nontrivial(self):
         """Nontrivial test of export_data functionality."""
         self.set_up_non_trivial()
+
+        # Feedback_thread_model setup here to access model for expected data.
```

Comment:
"We set up the feedback_thread_model here so that we can easily access it when computing the expected data later."